torch/__init__.py

```diff
@@ -444,6 +445,49 @@ def is_storage(obj):
     return type(obj) in _storage_classes
 
 
+_GLOBAL_DEVICE_CONTEXT = None
+
+def set_default_tensor_device(device):
```

Comment:
Why not just `set_default_device()` ?

Reply:
Happy to bikeshed it to that. It has tensor in the name because of `set_default_tensor_type`. But now that I look more carefully, I see it's `set_default_dtype`, so your name is better.